package.json

```diff
@@ -84,7 +84,6 @@
     "@commitlint/config-conventional": "^19.0.3",
     "@oclif/test": "^4.1.0",
     "@readme/better-ajv-errors": "^2.3.2",
-    "@readme/eslint-config": "^14.7.2",
```

Comment:
as part of this PR, i removed our eslint config and inlined our prettier setup so we're officially ✨ free of deprecation warnings ✨